portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsColorField.testcase

```diff
@@ -54,14 +54,53 @@ definition {
 	}
 
 	@description = "Verify that a Color Field can be duplicated"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test CanBeDuplicated {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-133135 CanBeDuplicated pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Color");
+
+		FormFields.editHelpTextField(fieldHelpText = "Help Text Test");
+
+		Form.gotoAdvancedTab();
+
+		FormViewBuilder.setFieldPredefinedDate(fieldValue = "FF0D0D");
```

Comment:
so maybe these macros FormViewBuilder.setFieldPredefinedDate, FormViewBuilder.duplicateField, FormViewBuilder.validateFieldLabel, also need to change the other one?

Reply:
@chaowang-123 they said we don`t need to worry about the PR`s we already sent, only to not use into the tickets we are doing from now on...that's why I thougth I shouldn`t put a new FormViewBuilder, but could keep the old ones, right?

Reply:
Sorry, I misunderstood their meaning again, sorry for the inconvenience caused to you

Reply:
@chaowang-123 no problem, don't worry! ;)